### DISCUSSION TITLE
Is it possible to create a pipeline without shaders in vulkan?

### DISCUSSION THREAD

**Discussion by Inori** [Inori/GPCS4#118] [Graphics]
Fox example, I just want to transform an image's layout.

**Comment by VelocityRa** [comment]
I would ask this in `#vulkan` in the Graphics Programming Discord: https://discord.gg/6mgNGk7.

**Comment by thomasgu112** [comment]
Maybe this could answer your question:
https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/chap10.html#pipelines-graphics
Nothing in the valid usage seems to imply it is necessary for the user to give any shaders when creating the pipeline.
